src/proxy.ts

```diff
@@ -1026,6 +1026,7 @@ if (settings.use_websocket) {
               } else if (msg.action === 'unsubscribe') {
                 logThis('User unsubscribed from confirmation: ' + remote_ip, log_levels.info)
                 tracking_db.get('users').find({ip: remote_ip}).assign({'tracked_accounts': []}).write()
+                wsSend(connection, {ack: 'unsubscribe', time: seconds, id: 'id' in msg ? msg.id : ""})
```

Comment:
Please add `const seconds = Math.round(Date.now() / 1000).toString()` before this line
Or you can't send seconds along with the response.

Reply:
Thanks; my bad, should've edited it in a proper editor instead of being lazy.